src/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -263,13 +263,9 @@ public boolean loadTownBlockList() {
 					world = getWorld(worldName);
 				}
 				File worldFolder = new File(dataFolderPath + File.separator + "townblocks" + File.separator + worldName);
-				File[] townBlockFiles = worldFolder.listFiles();
+				File[] townBlockFiles = worldFolder.listFiles((file)->file.getName().endsWith(".data"));
```

Comment:
Is this still skipping reading from the deleted folders inside of the townblocks\worldname\ folders?

Reply:
Yes, because the deleted folder doesn't end in `.data`

Reply:
A whole bunch of internet services are down because of a massive DDOS.